packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -296,16 +298,16 @@ class TimelineScreen extends Screen {
   }
 
   void _exportTimeline() {
-    // There will be a trailing comma on [debugTraceEvents]. Remove so that we
-    // do not have to deal with json formatting errors later.
-    final traceEvents = debugTraceEvents.isEmpty
-        ? ''
-        : debugTraceEvents.toString().replaceRange(
-            debugTraceEvents.length - 1, debugTraceEvents.length, '');
-    final String json = '{'
-        '"traceEvents":[$traceEvents],'
-        '"cpuProfile":[${debugCpuProfileResponse.toString()}]}';
-    downloadFile(json, 'timeline_${DateTime.now().toString()}.json');
+    final Map<String, dynamic> json = {
```

Comment:
add a TODO to add an analytics event for this.
Knowing how large the downloads are could help us understand how complex the problems are users are trying to solve are.

Reply:
done.